Doc/library/dis.rst

```diff
@@ -210,6 +214,11 @@ operation is being performed, so the intermediate analysis object isn't useful:
    This generator function uses the ``co_firstlineno`` and ``co_lnotab``
    attributes of the code object *code* to find the offsets which are starts of
    lines in the source code.  They are generated as ``(offset, lineno)`` pairs.
+   See :source:`Objects/lnotab_notes.txt` for the ``co_lnotab`` format and
+   how to decode it.
+
+   .. versionchanged:: 3.6
+      Line numbers can be not monotonically increasing.
```

Comment:
"not monotonically" is not easy to understand. In What's New in Python 3.6, I wrote:

"The format of the co_lnotab attribute of code objects changed to support a negative line number delta"